code/modules/mod/mod.dm

```diff
@@ -0,0 +1,541 @@
+	var/mob/living/carbon/carbon_user = user
+	if(!istype(carbon_user) || src != carbon_user.back)
+		return ..()
+	for(var/obj/item/part in mod_parts)
+		if(part.loc != src)
+			to_chat(carbon_user, "<span class='warning'>ERROR: At least one of the parts are still on your body, please retract them and try again.</span>")
+			playsound(src, 'sound/machines/scanbuzz.ogg', 25, FALSE)
+			return FALSE
+
+/obj/item/mod/control/MouseDrop(atom/over_object)
+	if(src != wearer?.back || !istype(over_object, /atom/movable/screen/inventory/hand))
+		return ..()
+	for(var/obj/item/part in mod_parts)
+		if(part.loc != src)
+			to_chat(wearer, "<span class='warning'>ERROR: At least one of the parts are still on your body, please retract them and try again.</span>")
+			playsound(src, 'sound/machines/scanbuzz.ogg', 25, FALSE)
+			return
+	if(!wearer.incapacitated())
+		var/atom/movable/screen/inventory/hand/ui_hand = over_object
+		if(wearer.putItemFromInventoryInHandIfPossible(src, ui_hand.held_index))
+			add_fingerprint(usr)
+			return ..()
+
+/obj/item/mod/control/attack_hand(mob/user)
+	if(seconds_electrified && cell.charge)
+		if(shock(user))
+			return
+	if(open && cell && loc == user)
+		to_chat(user, "<span class='notice'>You start removing [cell].</span>")
+		if(do_after(user, 50, target = src))
+			to_chat(user, "<span class='notice'>You remove [cell].</span>")
+			playsound(src, 'sound/machines/click.ogg', 50, TRUE)
+			if(!user.put_in_hands(cell))
+				cell.forceMove(drop_location())
+		return
+	return ..()
+
+/obj/item/mod/control/screwdriver_act(mob/living/user, obj/item/I)
+	if(..())
+		return TRUE
+	if(active || activating)
+		to_chat(user, "<span class='warning'>ERROR: Suit activated. Deactivate before further action.</span>")
+		playsound(src, 'sound/machines/scanbuzz.ogg', 25, TRUE)
+		return FALSE
+	to_chat(user, "<span class='notice'>You start to [open ? "screw the panel back on" : "unscrew the panel"]...</span>")
+	I.play_tool_sound(src, 100)
+	if(I.use_tool(src, user, 2 SECONDS))
+		if(active || activating)
+			return FALSE
+		I.play_tool_sound(src, 100)
+		user.visible_message("<span class='notice'>[user] [open ? "screws the panel back on" : "unscrews the panel"].</span>",
+			"<span class='notice'>You [open ? "screw the panel back on" : "unscrew the panel"].</span>",
+			"<span class='hear'>You hear metal noises.</span>")
+		open = !open
+	return TRUE
+
+/obj/item/mod/control/crowbar_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(!open)
+		to_chat(user, "<span class='warning'>ERROR: Suit panel not open.</span>")
+		playsound(src, 'sound/machines/scanbuzz.ogg', 25, TRUE)
+		return FALSE
+	if(modules.len)
+		for(var/obj/item/mod/module/module as anything in modules)
+			if(module.removable)
+				uninstall(module)
+				module.forceMove(drop_location())
+			else
+				audible_message("<span class='warning'>[src] indicates that [module] cannot be removed.</span>", "<span class='warning'>[src] flashes an error message.</span>")
```

Comment:
You want to give this a short range, otherwise everyone around you will hear it.

Reply:
Same with the others.